### ISSUE TITLE
Sky Void Additions/Additional Wandering Trades: Wandering Traders have no trade offers

### ISSUE THREAD

**Issue by yesitsye** [BPR02/SkyBlock_Collection#31]
Datapack version: 1.2.5
Minecraft version: 1.21.3

Wandering traders, that spawn while "Additional Wandering Trades" mechanic is turned on, are missing the Offers NBT tag and cannot be interacted with.

When Wandering Traders spawn this gets printed to console:
```
[20:29:44] [Server thread/WARN]: Failed to load offers: No key sell in MapLike[{maxUses:8,priceMultiplier:0.05f,rewardXp:1b,xp:1}]; No key buy in MapLike[{maxUses:8,priceMultiplier:0.05f,rewardXp:1b,xp:1}]
[20:29:44] [Server thread/WARN]: Failed to load offers: No key sell in MapLike[{maxUses:8,priceMultiplier:0.05f,rewardXp:1b,xp:1}]; No key buy in MapLike[{maxUses:8,priceMultiplier:0.05f,rewardXp:1b,xp:1}]
[20:29:44] [Server thread/WARN]: Failed to load offers: No key sell in MapLike[{maxUses:8,priceMultiplier:0.05f,rewardXp:1b,xp:1}]; No key buy in MapLike[{maxUses:8,priceMultiplier:0.05f,rewardXp:1b,xp:1}]
[20:29:44] [Server thread/WARN]: Failed to load offers: No key sell in MapLike[{maxUses:8,priceMultiplier:0.05f,rewardXp:1b,xp:1}]; No key buy in MapLike[{maxUses:8,priceMultiplier:0.05f,rewardXp:1b,xp:1}]
[20:29:44] [Server thread/WARN]: Failed to load offers: No key sell in MapLike[{maxUses:8,priceMultiplier:0.05f,rewardXp:1b,xp:1}]; No key buy in MapLike[{maxUses:8,priceMultiplier:0.05f,rewardXp:1b,xp:1}]
[20:29:44] [Server thread/WARN]: Failed to load offers: No key sell in MapLike[{maxUses:6,priceMultiplier:0.05f,rewardXp:1b,xp:1}]; No key buy in MapLike[{maxUses:6,priceMultiplier:0.05f,rewardXp:1b,xp:1}]
```

Additionally this error gets printed to console during datapack validation:
```
[20:28:03] [Worker-Main-4/ERROR]: Couldn't parse data file 'skyvoid_additions:trader_additions/use_4' from 'skyvoid_additions:predicate/trader_additions/use_4.json': DataResult.Error['Missing tag: 'skyvoid_additions:trader_additions/use_4' in 'minecraft:item'': fab[predicate=Optional[bw[entityType=Optional.empty, distanceToPlayer=Optional.empty, movement=Optional.empty, location=b[located=Optional.empty, steppingOn=Optional.empty, affectsMovement=Optional.empty], effects=Optional.empty, nbt=Optional.empty, flags=Optional.empty, equipment=Optional[bt[head=Optional.empty, chest=Optional.empty, legs=Optional.empty, feet=Optional.empty, body=Optional.empty, mainhand=Optional[cu[items=Optional.empty, count=d[min=Optional.empty, max=Optional.empty, minSq=Optional.empty, maxSq=Optional.empty], components=[], subPredicates={}]], offhand=Optional.empty]], subPredicate=Optional.empty, periodicTick=Optional.empty, vehicle=Optional.empty, passenger=Optional.empty, targetedEntity=Optional.empty, team=Optional.empty, slots=Optional.empty]], entityTarget=THIS]]
[20:28:03] [Render thread/INFO]: Loaded 1337 recipes
[20:28:03] [Render thread/ERROR]: Failed to load function skyvoid_additions:mechanics/trader_additions/special_trade
java.util.concurrent.CompletionException: java.lang.IllegalArgumentException: Whilst parsing command on line 9: Failed to parse structure: Failed to get element ResourceKey[minecraft:predicate / skyvoid_additions:trader_additions/use_4] at position 150: ...ions/use_4<--[HERE]
	at java.base/java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:315) ~[?:?]
	at java.base/java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:320) ~[?:?]
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1770) ~[?:?]
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.exec(CompletableFuture.java:1760) ~[?:?]
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:387) ~[?:?]
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1312) ~[?:?]
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1843) ~[?:?]
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1808) ~[?:?]
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:188) ~[?:?]
Caused by: java.lang.IllegalArgumentException: Whilst parsing command on line 9: Failed to parse structure: Failed to get element ResourceKey[minecraft:predicate / skyvoid_additions:trader_additions/use_4] at position 150: ...ions/use_4<--[HERE]
	at ik.a(SourceFile:80) ~[minecraft-1.21.3-client.jar:?]
	at amn.a(SourceFile:87) ~[minecraft-1.21.3-client.jar:?]
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768) ~[?:?]
	... 6 more
[20:28:03] [Render thread/ERROR]: Failed to load function skyvoid_additions:mechanics/trader_additions/normal_trade
java.util.concurrent.CompletionException: java.lang.IllegalArgumentException: Whilst parsing command on line 9: Failed to parse structure: Failed to get element ResourceKey[minecraft:predicate / skyvoid_additions:trader_additions/use_4] at position 150: ...ions/use_4<--[HERE]
	at java.base/java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:315) ~[?:?]
	at java.base/java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:320) ~[?:?]
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1770) ~[?:?]
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.exec(CompletableFuture.java:1760) ~[?:?]
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:387) ~[?:?]
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1312) ~[?:?]
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1843) ~[?:?]
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1808) ~[?:?]
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:188) ~[?:?]
Caused by: java.lang.IllegalArgumentException: Whilst parsing command on line 9: Failed to parse structure: Failed to get element ResourceKey[minecraft:predicate / skyvoid_additions:trader_additions/use_4] at position 150: ...ions/use_4<--[HERE]
	at ik.a(SourceFile:80) ~[minecraft-1.21.3-client.jar:?]
	at amn.a(SourceFile:87) ~[minecraft-1.21.3-client.jar:?]
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768) ~[?:?]
	... 6 more
```

**Comment by BPR02** [comment]
Can confirm. Thanks for the bug report. Working on a fix right now.

**Comment by BPR02** [comment]
Fixed with Sky Void Additions v1.2.6
